homeassistant/components/anthemav/strings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        }
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "cannot_receive_deviceinfo": "Failed to retreive MAC Address. Make sure the device is turned on"
```

Comment:
Can we really connect but fail to retrieve the mac address if the device is turned off?